Explorer/Assets/DCL/Multiplayer/Movement/Systems/SDKAvatarShapesMotionSystem.cs

```diff
@@ -0,0 +1,108 @@
+using Arch.Core;
+using Arch.System;
+using Arch.SystemGroups;
+using Arch.SystemGroups.DefaultSystemGroups;
+using DCL.AvatarRendering.AvatarShape.UnityInterface;
+using DCL.Character.Components;
+using DCL.CharacterMotion.Animation;
+using DCL.CharacterMotion.Components;
+using DCL.Diagnostics;
+using DCL.Profiles;
+using ECS.Abstract;
+using UnityEngine;
+
+namespace DCL.Multiplayer.Movement.Systems
+{
+    [UpdateInGroup(typeof(PresentationSystemGroup))]
+    [LogCategory(ReportCategory.AVATAR)]
+    public partial class SDKAvatarShapesMotionSystem : BaseUnityLoopSystem
+    {
+        private const float ROTATION_SPEED = 5f;
+        private const float RUN_SPEED_THRESHOLD = 9.5f;
+        private const float JOG_SPEED_THRESHOLD = 4f;
+
+        private SDKAvatarShapesMotionSystem(World world) : base(world) { }
+
+        protected override void Update(float t)
+        {
+            UpdateMotionQuery(World);
+        }
+
+        [Query]
+        [None(typeof(Profile))]
+        private void UpdateMotion(
+            in IAvatarView view,
+            ref CharacterTransform characterTransformComponent,
+            ref CharacterLastPosition characterLastPositionComponent,
+            ref CharacterAnimationComponent animationComponent)
+        {
```

Comment:
do you really need this None `[None(typeof(Profile))]`, since you already have `CharacterLastPosition ` that is only in your sdk-scene avatars?

Reply:
and did you try to use `[All(typeof(SDKAvatarShapeComponent))]`. That would be a good hint to show it works only on scene avatars

Reply:
Done: https://github.com/decentraland/unity-explorer/pull/3297/commits/2c52f1cf6deb762cbb1e77e2e9141878a0f7840f